openapi.yaml

```diff
@@ -16,175 +16,218 @@ host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/notes-api
 produces:
   - application/json
 paths:
-  /pets:
+  /notes:
     get:
-      summary: List all pets
+      summary: Get notes for a student
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - notes
+      operationId: getNotes
       parameters:
-        - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
-        - in: query
-          name: species
-          description: Search by pet species
+        - name: studentID
+          in: query
+          type: string
+          required: true
+          description: 9-digit OSU ID of student whose notes will be retrieved
+        - name: q
+          in: query
+          type: string
           required: false
+          description: Query string to be used when searching through the contents of each note
+        - name: sortKey
+          in: query
           type: string
+          enum:
+            - 'lastModified'
+            - 'source'
+            - 'permissions'
+            - 'contextType'
```

Comment:
Just a style nitpick, sometimes the string value was quoted with single quotes but sometimes doesn't. I know yaml handles this pretty well but just wanna make the code looks consistent.

Reply:
Ok I fixed the other `enum` to be consistent.